test/EasyRdf/Literal/BooleanTest.php

```diff
@@ -67,7 +67,7 @@ public function testConstructString1()
     {
         $literal = new Boolean('1');
         $this->assertStringEquals('1', $literal);
-        $this->assertInternalType('bool', $literal->getValue());
+        $this->assertEquals('boolean', gettype($literal->getValue()));
```

Comment:
```suggestion
        $this->assertIsBool($literal->getValue());
```
https://phpunit.readthedocs.io/en/9.3/assertions.html#assertisbool

Reply:
I had no problems with PHPUnit 7, ref: https://travis-ci.com/github/easyrdf/easyrdf/jobs/452308483#L885

But I had to change the first parameter from `bool` to `boolean`. Was that wrong?

Reply:
Oh sorry, I meant do not apply my suggestion because it is not supported on phpunit 7.x series

Reply:
I am not sure what you mean. So you still suggest to use `assertIsBool`?

Reply:
No no :)
If (weDropPhpUnit7) {
applyMyComment
} else {
// ignore it
}

Reply:
:)